internal/config/config.go

```diff
@@ -59,8 +61,21 @@ type Settings struct {
 }
 
 func defaultConfig(dataDir, configFile string) *Config {
+	forcePrepend := forcePrependDefault
+	forcePrependEnv := os.Getenv("ASDF_FORCE_PREPEND")
+	switch forcePrependEnv {
+	case "":
+		forcePrepend = forcePrependDefault
+		if runtime.GOOS == "darwin" {
+			forcePrepend = true
+		}
+	case "yes":
+		forcePrepend = true
+	default:
+		forcePrepend = false
+	}
```

Comment:
I think the use of switch here adds unnecessary complexity. For instance, there's not longer any case where the value of `forcePrependDefault` is used since its overwritten by the default case.

I suggest using an if instead:

```
	forcePrependEnv := os.Getenv("ASDF_FORCE_PREPEND")
	if forcePrependEnv == "yes" || (forcePrependEnv == "" && runtime.GOOS == "darwin") {
		forcePrepend = true
	}
```

Reply:
Yes. that's exactly it!
Thank you!

https://github.com/asdf-vm/asdf/pull/2011/commits/d76ef9b13754529d203389080465b2007f050110